### PR TITLE
Eshaben/update precompiles

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR addresses a few items:
-  general clean up for consistency throughout the docs
-  adds a new pallets section (combining pallets & precompiles didn't seem like the best idea, especially for example the staking pallet/precompile as that page would have been extremely long and there are some cases where there are pallets and no precompiles and vice versa)
- updates all of the precompile pages for consistency in format

Image paths need to be updated still, but I think it might be easier to do one clean sweep of updating the images after the translations are done

⭕️ Will complete the checklist after an initial review ⭕️

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [x] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Mkdocs redirects: https://github.com/PureStake/moonbeam-mkdocs/pull/85
Chinese image paths: https://github.com/PureStake/moonbeam-docs-cn/pull/128

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [x] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

- Need to remove the old index page icons for precompiles
- Need to add redirects (you can use the same ones from the mkdocs pr: https://github.com/PureStake/moonbeam-mkdocs/pull/85)

### CODE REVIEW

**Comment by albertov19** [comment]
@eshaben can you fix conflicts plz

**Comment by eshaben** [comment]
@albertov19 fixed! ✅

**Comment by eshaben** [comment]
@albertov19 this is ready to be reviewed. i didn't address the utility page being "half-done" because I don't know what else to add to it. I think it makes sense to keep it the way it is for consistency but definitely open to suggestions! 🙂

File: builders/pallets-precompiles/pallets/utility.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# The Utility Pallet
+
+![Utility Moonbeam Banner](/images/builders/pallets-precompiles/pallets/utility-banner.png)
+
+## Introduction {: #introduction }
+
+Through Substrate's utility pallet, users on Moonbeam can include multiple calls into a single transaction and use derivative accounts to send calls.
+
+Some of the functionality of the utility pallet is also available through a batch precompile. The precompile is a Solidity interface that enables you to perform batch calls using the Ethereum API. Please refer to the [Batch Precompile](/builders/build/precompiles/batch){target=_blank} guide for more information.
```
Comment (albertov19) [inline builders/pallets-precompiles/pallets/utility.md:15]: Remember what I mentioned that the batch precompile has nothing to do with the `utility` pallet

File: builders/pallets-precompiles/pallets/utility.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Utility Pallet
+description: Learn about the available extrinsics for the utility pallet on Moonbeam and how to interact with them using Polkadot.js Apps and the Polkadot.js API.
+keywords: utility, batch, substrate, pallet, moonbeam, polkadot
+---
+
+# The Utility Pallet
```
Comment (albertov19) [inline builders/pallets-precompiles/pallets/utility.md:7]: I wonder if we should add examples or something as this feels a bit empty. We could:

1. Explain a bit more derivative accounts (check the xcm-transactor page-> http://127.0.0.1:8000/builders/xcm/xcm-transactor/#general-xcm-definitions) We could then add a reference from the xcm-transactor page to here
2. Do an example with the batching?

**Review by albertov19 — CHANGES_REQUESTED** [review]
Thanks for doing this and nitpicking (standardizing) lots of these pages. My main feedback is regarding the utility page, let me know what you think

**Comment by albertov19** [comment]
Ah also, check for warnings when building the page, I got a bunch of redirects not existing warnings

**Comment by eshaben** [comment]
> Ah also, check for warnings when building the page, I got a bunch of redirects not existing warnings

Yes thank you, need to still create PR for mkdocs and chinese repo to reflect changes made here

**Comment by eshaben** [comment]
@albertov19 this is ready again

**Review by albertov19 — APPROVED** [review]

**Comment by albertov19** [comment]
Looks good!!! 😎 🚀

**Pull request merged**